SmurfTestingScripts/smurf_compare_scenarios.py

```
@@ -73,15 +102,20 @@ def run_scenarios():
+            test_results.append(TestResult(scenario["name"], False, None))
             return False
-        print("\n\033[92m..... Done\033[0m")
+        print(f"\n\033[92m..... Test case: {scenario['name']} finished\033[0m")
 
+        etalons_equal = None
         if not args.create_etalons:
-            if not compare_output(scenario["name"]):
+            etalons_equal = compare_output(scenario["name"])
+            if not etalons_equal:
                 tests_passed = False
+        test_results.append(TestResult(scenario["name"], False if commands_ok.value == 0 else True, etalons_equal))
```

Ruff (0.11.9)

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)